iib/web/models.py

```diff
@@ -1398,3 +1407,28 @@ def validate_request_params(request_params, required_params, optional_params):
             and not request_params[param]
         ):
             del request_params[param]
+
+
+def validate_registry_auths(registry_auths):
+    """
+    Validate registry_auths for a build request.
+
+    Only auth item in dockerconfig.json is supported for iib.
+
+    :param dict registry_auths: User provided dockerconfig for authentication
+      to private registries
+    :raises ValidationError: if registry_auths are not in valid format
+    """
+    auths = 'auths'
+    if not isinstance(registry_auths, dict):
+        raise ValidationError('"registry_auths" must be a dict')
+    if list(registry_auths.keys()) != [auths]:
+        raise ValidationError(f'"registry_auths" must contain single key "{auths}"')
+    if not registry_auths[auths] or not isinstance(registry_auths[auths], dict):
+        raise ValidationError(f'"registry_auths.{auths}" must be a non-empty dict')
+    for reg, auth_dict in registry_auths[auths].items():
+        err_msg = f'{reg} in registry_auths has auth value in incorrect format'
```

Comment:
How about changing this to `{reg} in registry_auths has auth value in an incorrect format. It should be a base64 encoded token`?

Reply:
It's more complex than that. It's well described in the API docs. How about something like `{reg} in registry_auths has auth value in an incorrect format. See the API docs for details on the expected format` ?

Reply:
I added second option `See the API docs for details on the` ..